Feature/add images (#26)
CitrineInformatics/gemd-docs: docs/specification/attribute-templates.md

If the value is not compatible with the template (e.g. if the template specifies that
the value must be in the range [0, 1] but the value is 2), then the data are invalid.

> ![attribute templates](img/attribute-templates.png)

Reviewer: This image as the "Confidential" footer in it.  I assume it is not confidential, and would suggest cropping it.

`description` | String | None | Some text describing what this template is
`tags`        | Set[String]| Empty set | [Tags](../tags)

Attribute Templates are critical in defining which quantities are comparable or equivalent.
When Attributes are added to core objects (Runs or Specs), they should be associated with

> an appropriate Attribute Template. This also determines how Values are collected and aggregated

Reviewer: ```suggestion
an appropriate Attribute Template.
This association is meant to inform how Values are collected and aggregated
```
Newline added per sentence (markdown ignores single newlines just like latex).

> in Tables.

Reviewer: ```suggestion
in downstream consumers of GEMD data.
```
More generic.
